frontend/src/components/Pages/FrontPage/RobotCards/PressureStatusView.tsx

```diff
@@ -13,16 +14,47 @@ const StyledTopography = styled(Typography)<{ $fontSize?: 24 | 16 | 18 | 32 | 40
 `
 export interface PressureStatusViewProps {
     pressure?: number
+    pressureStatus?: PressureStatus
     itemSize?: 24 | 16 | 18 | 32 | 40 | 48 | undefined
+    upperPressureWarningThreshold: number
+    lowerPressureWarningThreshold: number
 }
 
-const PressureStatusView = ({ pressure, itemSize }: PressureStatusViewProps): JSX.Element => {
+const PressureStatusView = ({
+    pressure,
+    pressureStatus,
+    itemSize,
+    upperPressureWarningThreshold,
+    lowerPressureWarningThreshold,
+}: PressureStatusViewProps): JSX.Element => {
     const barToMillibar = 1000
     let icon_color: string = tokens.colors.interactive.primary__resting.hex
 
-    if (!pressure) {
+    if (!pressure || !upperPressureWarningThreshold || !lowerPressureWarningThreshold) {
```

Comment:
```suggestion
    if (!pressureInBar || !upperPressureWarningThreshold || !lowerPressureWarningThreshold) {
```

Reply:
If the lowerPressureWarningThreshold and upperPressureWarningThreshold are required this is not necessary.